TesseractOCR/build_dependencies.sh

```diff
@@ -7,8 +7,9 @@
 
 GLOBAL_OUTDIR="`pwd`/build"
 LOCAL_OUTDIR="./outdir"
-LEPTON_LIB="`pwd`/leptonica-1.71"
 TESSERACT_LIB="`pwd`/tesseract-3.03"
+LEPTON_LIB="`pwd`/leptonica-1.72"
+IMAGE_LIBS="`pwd`/libtiff-ios-master/dependencies"
```

Comment:
Ok. I'll try to add README as a part of the commit.
I'll also try to setup pod file and include it in this commit.
Finally, tesseract is available as a git repo, so we are able to refer to any of its commits from the history. I think we also can find leptonica repo.